components/ProfilePersonalInfo.js

```diff
@@ -33,35 +31,44 @@ const ProfilePersonalInfo = () => {
         renderItem={renderPersonalInfoItem}
       />
       <View style={styles.buttonContainer}>
-        <TouchableOpacity style={styles.button}>
-          <Text style={styles.buttonText}>Settings</Text>
-        </TouchableOpacity>
-        {user && <SignOutButton onPress={handleSignOut} />}
+        <Modal
+          animationType="slide"
+          transparent={true}
+          visible={modalVisible}
+          onRequestClose={() => {
+            toggleModal();
+          }}
+        >
+          <TouchableWithoutFeedback onPress={() => toggleModal()}>
+            <View style={styles.modalOverlay} />
+          </TouchableWithoutFeedback>
+        </Modal>
       </View>
     </View>
   );
 };
 
 const styles = StyleSheet.create({
```

Comment:
This should be in a .cs file to keep consistency

Reply:
Or am I wrong? I guess I haven't been paying attention to mobile

Reply:
tf is a .cs file? I haven't been doing this for any modals.